js/index.js

```diff
@@ -1,7 +1,49 @@
+/** @type {HTMLCanvasElement} */
+const canvas = document.getElementById("canvas");
+
+const ctx = canvas.getContext("2d");
+
+let backgroundImage = new Image()
+backgroundImage.src = "images/road.png"
+
+let player = new Component(50,90,canvas.width / 2 - 25, 400, "", ctx)
+const game = new Game(ctx, canvas.width, canvas.height, player)
 window.onload = () => {
+  
   document.getElementById('start-button').onclick = () => {
+    
     startGame();
+    
+    
   };
+  
+  function startGame() {
+    canvas.style.backgroundImage = "url('../images/road.png')"
```

Comment:
Another way to do that is creating the image inside of Game's constructor.

```
const backgroundImage = new Image();
backgroundImage.src = "../images/road.png"
this.imgRoad = backgroundImage;
```
and then drawing it inside of the update function

`this.ctx.drawImage(this.imgRoad, 0, 0, canvas.width, canvas.height)`